### PR TITLE
add issue template of question

### LINKED ISSUE #671: Add Issue template about questions
**What would you like to be added**:
Add Issue template about questions.

**Why is this needed**:
When needing to ask question about karmada, I want to have a issue template to fill.

### PR DESCRIPTION
**Pull request by lonelyCZ**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Add issue template of question.

**Which issue(s) this PR fixes**:
Fixes #671 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+<!-- Please use this template while providing as much info as possible. Thanks!-->
+**What happened and what you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+**Anything else we need to know?**:
+
+**Environment**:
+- Karmada version:
```
Comment (pigletfly) [inline .github/ISSUE_TEMPLATE/question.md:16]: should we provide k8s version as well?

**Review by pigletfly — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/question.md (hunk 1 above)
Comment (lonelyCZ) [reply .github/ISSUE_TEMPLATE/question.md:16]: > should we provide k8s version as well?

yes

File: .github/ISSUE_TEMPLATE/question.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+labels: kind/question
+
+---
+
+<!-- Please use this template while providing as much info as possible. Thanks!-->
+**What happened and what you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+**Anything else we need to know?**:
```
Comment (RainbowMango) [inline .github/ISSUE_TEMPLATE/question.md:13]: Same with the `kind/bug` template? I don't think so.

File: .github/ISSUE_TEMPLATE/question.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+name: Question
+about: Question relating to Karmada.
+labels: kind/question
+
+---
+
+<!-- Please use this template while providing as much info as possible. Thanks!-->
```
Comment (RainbowMango) [inline .github/ISSUE_TEMPLATE/question.md:8]:
```suggestion
<!-- Please use this template while asking a new question. Thanks!
Please make sure you have read the FAQ and searched the issue list.
- FAQ: https://github.com/karmada-io/karmada/blob/master/docs/frequently-asked-questions.md
- Issues: https://github.com/karmada-io/karmada/issues?q=is%3Aissue+
-->
```

File: .github/ISSUE_TEMPLATE/question.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+**Please provide an in-depth description of the question you have**:
+
+**What do you think about this question?**:
+
+**How to reproduce it**:
+
```
Comment (RainbowMango) [inline .github/ISSUE_TEMPLATE/question.md:14]: I suggest removing this item.

**Comment by XiShanYongYe-Chang** [comment]
hi, @lonelyCZ , please squash you commit.

**Comment by RainbowMango** [comment]
Yeah. Squash your commit to one will be all set. Thanks.

**Review by lonelyCZ — COMMENTED** [review]
> Yeah. Squash your commit to one will be all set. Thanks.

Ok, I try to solve it immediately.

**Comment by XiShanYongYe-Chang** [comment]
@lonelyCZ  Why close this MR?

**Comment by RainbowMango** [comment]
Do you need help?

**Comment by lonelyCZ** [comment]
@RainbowMango @XiShanYongYe-Chang 
I just trid to squash commits, but there are some conflicts. Then I delete the branch and want to recreate it.

I need learn how to squash commits correctly.

**Comment by RainbowMango** [comment]
I guess you have messed your branch. So, my suggestion would be :
- Copy `.github/ISSUE_TEMPLATE/question.md ` locally
- Remove the branch `lonelyCZ-patch-1` from your workspace and your forked repo.
- git checkout -b lonelyCZ-patch-1
- Copy `.github/ISSUE_TEMPLATE/question.md ` back
- git commit -s -m "xxx"
- git push origin lonelyCZ-patch-1 -f

**Comment by lonelyCZ** [comment]
@RainbowMango  Thank you, I try it now.

**Comment by lonelyCZ** [comment]
@RainbowMango 
I can`t reopen this pr, should I recreate a pr?

**Comment by RainbowMango** [comment]
Yes, that'll be much easier.

**Comment by lonelyCZ** [comment]
@RainbowMango @XiShanYongYe-Chang @pigletfly 

Thank you very much. I have more knowledge to learn.

**Pull request closed without merging**